MutableStorageKVP/.gitignore

```diff
@@ -0,0 +1,353 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
Is this file meant to be part of this project? Most of the ignores here are irrelevant.

Reply:
removed.

Reply:
removed